logical/plugin/backend_client.go

```diff
@@ -11,9 +11,8 @@ import (
 // backendPluginClient implements logical.Backend and is the
 // go-plugin client.
 type backendPluginClient struct {
-	broker       *plugin.MuxBroker
-	client       *rpc.Client
-	pluginClient *plugin.Client
+	broker *plugin.MuxBroker
+	client *rpc.Client
 
 	system logical.SystemView
 	logger log.Logger
```

Comment:
@briankassouf Since the logger never really goes over the wire, I think it's safe to remove the shim in the `Setup()` func and `logical/plugin/logger.go` file. What do you think? I believe the same would apply to the system shim.